torch/_prims/context.py

```diff
@@ -129,6 +129,8 @@ def __torch_function__(
             func = torch._decomp.decomposition_table.get(orig_func, None)
         elif func is None and isinstance(orig_func, torch._ops.OpOverloadPacket):
             default = getattr(orig_func, "default", None)
+            if default is None:
+                default = getattr(orig_func, orig_func._dir[0], None)
```

Comment:
This needs to be guarded since there isn't any guarantee that `orig_func._dir` is not empty. If that access is out of bounds then default should remain `None`. As written it would raise an exception.

Reply:
Urg, fixed, good catch.